tests/unit/lms/services/vitalsource/_client_test.py

```diff
@@ -123,63 +123,6 @@ def test_get_table_of_contents_error(self, client, http_service, status_code):
         with pytest.raises(ExternalRequestError):
             client.get_table_of_contents("BOOK_ID")
 
-    @pytest.mark.parametrize(
-        "response_xml",
-        (
-            # There are many more attributes in real results than these, but at the
-            # moment we don't really care about any of them.
-            """<?xml version="1.0" encoding="UTF-8"?>
-                <licenses>
-                    <license imprint="IMPRINT" name="NAME"/>
-                </licenses>
-            """,
-            """<?xml version="1.0" encoding="UTF-8"?>
-                <licenses>
-                    <license imprint="IMPRINT" name="NAME"/>
-                    <license imprint="IMPRINT2" name="NAME2"/>
-                </licenses>
-            """,
-        ),
```

Comment:
This area here exercised a piece of code which checks for single or multiple items. Removing this caused a coverage gap.